pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -34,8 +34,10 @@ const _ = cockpit.gettext;
 // Found in /usr/share/crypto-policies/policies/
 const cryptopolicies = {
     DEFAULT: _("Recommended, secure settings for current threat models."),
+    "DEFAULT:SHA1": _("DEFAULT with SHA-1 signature verification allowed"),
     FUTURE: _("Protects from anticipated near-term future attacks at the expense of interoperability."),
```

Comment:
We talked that LEGACY should be last in the list and probably marked as "testing" or something similar as it is actually not advised to use in production but is meant for testing.

Reply:
Wasn't that about `FUTURE`?

Reply:
It was, sorry, meant FUTURE